src/modules/common/components/__tests__/__snapshots__/Page.spec.js.snap

```diff
@@ -19,7 +19,7 @@ exports[`Page should render 1`] = `
   />
   <Translate(LastUpdateInfo)
     language="en"
-    lastUpdate={"2016-01-07T09:36:24.000Z"}
+    lastUpdate={"2016-01-07T08:36:24.000Z"}
```

Comment:
We need to do something about this :) I'll create an other task for this. So you do no need to change anything here.

The reason why these changes were necessary *for you* is the difference in the timezone. We need to switch to UTC here.